tests/e2e/testfiles/values.yaml

```diff
@@ -1,4 +1,19 @@
 # List of E2E Tests Scenarios
+- scenario: "Install Authorization Proxy Server"
+  paths:
+    - "testfiles/authorization-templates/csm_authorization_proxy_server.yaml"
+  modules:
+    - "authorizationproxyserver"
+    - "authorization"
```

Comment:
This is the sidecar value. Since no driver is being installed, it's not required here

Reply:
Removed sidecar.